notification_controller/controller/controller.go

```diff
@@ -38,8 +39,8 @@ var (
 	appProjects  = schema.GroupVersionResource{Group: application.Group, Version: "v1alpha1", Resource: application.AppProjectPlural}
 )
 
-func newAppProjClient(client dynamic.Interface, namespace string) dynamic.ResourceInterface {
-	resClient := client.Resource(appProjects).Namespace(namespace)
```

Comment:
Is this necessary? AppProjects are still restricted to only living in the controller namespace.